srcpkgs/util-linux/template

```diff
@@ -1,8 +1,8 @@
 # Template file for 'util-linux'
 # Keep this package sync with util-linux-libs
 pkgname=util-linux
-version=2.35.1
```

Comment:
There is a note in this file:

    # Keep this package sync with util-linux-libs